src/tools/rustc-std-workspace-alloc/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![feature(no_core)]
 #![no_core]
+#![deny(rust_2018_idioms)]
```

Comment:
This is unnecessary (already enforced by rustbuild).

Reply:
Added it for consistency with `src/tools/rustc-std-workspace-core/lib.rs`. Keep both or remove both?

Reply:
We can remove both.

Reply:
FWIW, the workspace-core one was removed in https://github.com/rust-lang/rust/pull/62910, you probably have outdated sources.

Reply:
Yeah. Updated.